org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/servicetemplates/topologytemplates/TopologyTemplateResource.java

```diff
@@ -1,18 +1,18 @@
 /*******************************************************************************
  * Copyright (c) 2012-2017 University of Stuttgart.
  * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v2.0
+ * are made available under the terms of the Eclipse Public License v1.0
  * and the Apache License 2.0 which both accompany this distribution,
- * and are available at http://www.eclipse.org/legal/epl-v20.html
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
  * and http://www.apache.org/licenses/LICENSE-2.0
  *
  * Contributors:
  *     Oliver Kopp - initial API and implementation
  *     Karoline Saatkamp - add split API
- *     Philipp Meyer - support for source directory
  *******************************************************************************/
-package org.eclipse.winery.repository.rest.resources.servicetemplates.topologytemplates;
```

Comment:
Please do not alter the package. The REST resources are no contained in org.eclipse.winery.repository.rest.resources and not in org.eclipse.winery.repository.resources any more. Please port your changes to the new package.